src/cf-forms/src/atoms/label.less

```diff
@@ -5,6 +5,14 @@
         .u-webfont-regular();
         color: @label-helper;
         font-size: unit( @size-v / @base-font-size-px, em );
+
+        &__block {
+            display: block;
+        }
+    }
+
+    & > &_helper__block {
+        margin-top: unit( 10px / @base-font-size-px, em );
```

Comment:
Why not put this in with the display rule?

Reply:
I was ensuring that the margin is *only* applied when the helper is inside a label. 
It should always be so I guess they can be combined.

Reply:
Fixed in https://github.com/cfpb/capital-framework/pull/632/commits/147df19d09ea11c0d98c83d8f1d9488908510bb8

Reply:
Yeah, since it's an `_element` it should never be used outside the `a-label` parent